cfpb/consumerfinance.gov
# Add static html list to raf template for no-js visitors

Adds static html to the raf-tool which is displayed when javascript is not supported or enabled.

Testing:
 - Pull branch
 - Visit http://localhost:8000/coronavirus/mortgage-and-housing-assistance/renter-protections/find-help-with-rent-and-utilities/
 - Things should work as normal
 - Open devtools and use cmd + shift +p to open the `run` panel, type `Disable Javascript` and run that command when it populates
 - Refresh the page
 - The static html list should be rendered
